extensions/amp-story/1.0/amp-story.css

```diff
@@ -913,3 +915,12 @@ amp-story .amp-video-eq,
 .i-amphtml-story-interactive-component[chip-style="shadow"][theme="dark"] {
   --interactive-chip-shadow: 0px 2px 8px 0px rgba(0, 0, 0, 0.3) !important;
 }
+
+/* Properties locked by specific components */
+amp-story-interactive-quiz {
+	--interactive-background-color: white !important
+}
+
+amp-story-interactive-quiz[theme="dark"] {
+	--interactive-background-color: #202124 !important;
+}
```

Comment:
The quizzes (and in the future multi-option polls) won't allow changing the option-container background color (stays white or dark-gray), so they are marked important here.